src/schema/artwork/index.js

```diff
@@ -123,6 +124,24 @@ export const artworkFields = () => {
     availability: {
       type: GraphQLString,
     },
+    bidderStatus: {
```

Comment:
Actually question-- why use the `bidderStatus` nomenclature here instead of `lotStanding`?

Reply:
I think it's just me inventing terms in reaction static comp. The bidder stats is what we wanted to use there - I named it that in absence of knowing real terms for all things auctions. If it sounds off - totally up for changing it. In my mind `BidderStatus` showed that we are getting the data for a given user standings on a particular lot. Where `lotStanding` does not signal that it's a specific user data.

Reply:
My naive mind agrees with @oxaudo, but we’re not using this in production yet, so we could still change if you think that’s appropriate?

Reply:
I might tend towards `lotStanding` since it's what we use elsewhere (`bidderStatus` can be misleading as to me it would refer to the status of my `bidder` meaning, am I qualified or not to bid on this auction).

Reply:
Got it. @oxaudo do you want to take this?

Reply:
Sure. Will update a bit later. Thanks @sweir27 for flagging... The only thing I think it'll be cool to know from the name that it's a current user info. Mayne it can be `currentUserLotStanding` - or it's too verbose and lotStanding applies that it's a current user specific info?

In my mind just `lotStanding` will apply that this field contains the most current info from all the bidders where in this case it's specifically scoped to the user on this sale artwork information. @sweir27 ?

Reply:
Thus far we have been referring to the ‘current user’ as ‘me’ in MP, so (shudder) it should probably be `myLotStandings`